src/api/app/controllers/webui/groups/bs_requests_controller.rb

```diff
@@ -28,6 +42,29 @@ def set_group
       def request_method
         REQUEST_METHODS[params[:dataTableId]]
       end
+
+      def set_bs_request
+        return unless Flipper.enabled?(:request_index, User.session)
+
+        @bs_requests = @user_or_group.requests
+      end
+
+      def filter_involvement
+        return if params[:involvement]&.compact_blank.blank?
+
+        @selected_filter['involvement'] = params[:involvement]
+
+        @bs_requests = case
+                       when @selected_filter['involvement'].include?('incoming')
+                         BsRequest::FindFor::Query.new({ group: @user_or_group.title, roles: [:maintainer] }, @bs_requests).all
```

Comment:
@danidoni with the latest development of refactoring the `BsRequest::FindFor::Query` out in the other controllers, was there any agreement of keep using it here or also get rid of it?

Reply:
Let me take a look

Reply:
I did like @hennevogel said and used code from https://github.com/openSUSE/open-build-service/blob/a4e8dc6e77155d2b27306b9e5f7db444be8f413c/src/api/app/controllers/webui/users/bs_requests_controller.rb#L50

Reply:
:+1: